### ISSUE TITLE
please add a way to donate

### ISSUE THREAD

**Issue by justTravis** [zocker-160/keyboard-center#45]
After suffering a couple of weeks without this app, telling myself that I could learn to live without my G910 keys.  And that I will sooner or later have to, and now is the time....

I finally asked here about a fix and within HOURS I had a response.  And it feels so GOOD to have my keys back.

I'd love to be able to "buy you a coffee" or whatever you want to call it.

**Comment by Refrained-LM** [comment]
I agree with JustTravis  I would like the ability to give you something for your hard work. I'm able to game comfortably with this app you created. (well at the moment it's not working but I have been using this app for over a year plus with no problems)
